src/properties.c

```diff
@@ -604,7 +604,7 @@ int properties_create_device(struct knot_thing *thing,
 
 	rc = set_rabbit_mq_url(thing, conf_files->rabbitmq_path);
```

Comment:
```suggestion
	rc = set_rabbit_mq_url(thing, conf_files->cloud_path);
```

I believe that could_path should be used here, since we don't have rabbitmq_path anymore.